README.md

```diff
@@ -6,13 +6,25 @@ A few gtk-rs examples. To build, just do:
 > cargo build
 ```
 
-or to enable GTK 3.10 examples as well:
+or to enable GTK 3.1x, 3.2x, all examples as well:
 
 ```Shell
-> cargo build --features gtk_3_10
+> cargo build --features gtk_3_18
+> cargo build --features gtk_3_22_30
+> cargo build --all-features
 ```
 
-And then run the executables. Please be sure to have installed all the required libraries before building examples (the list is available on [gtk](https://github.com/gtk-rs/gtk/).
+And then run the executables with:
+
+``` Shell
+./target/debug/EXAMPLE-NAME
+```
+or with cargo run (repeating the compilation features used above), example:
+``` Shell
+cargo run --all-features --bin EXAMPLE-NAME
+```
+
+Please be sure to have installed all the required libraries before building examples (the list is available on [gtk](https://github.com/gtk-rs/gtk/).
```

Comment:
There is a missing closing paren at the end. :)

Reply:
Also, I'd replace "\[gtk\]" with "\[gtk-rs\]" to make sure people don't confuse it with GTK official repository.